src/sonic-supervisord-utilities-rs/src/bin/supervisor_proc_exit_listener.rs

```diff
@@ -0,0 +1,11 @@
+//! Supervisor process exit listener binary
+
+use sonic_supervisord_utilities_rs::supervisor_proc_exit_listener::main as supervisor_main;
+use std::process;
+
+fn main() {
+    if let Err(e) = supervisor_main() {
+        eprintln!("Error: {}", e);
+        process::exit(1);
+    }
```

Comment:
You can just have `main()` return a result. See https://doc.rust-lang.org/rust-by-example/error/result.html#using-result-in-main.